### PR TITLE
[#1] Refactor submodules to match type-natural

### PR DESCRIPTION
**Pull request by pgujjula**
The code is refactored to according to the layout specified in issue #1. Other small improvements include
* adding the default stylish-haskell.yaml
* bumping the nightly resolver to use ghc-8.10
* removing the unnecessary executable files included in the default stack project
* converting the cabal file to package.yaml

### CODE REVIEW

File: .stylish-haskell.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  - NoImplicitPrelude
+  - OverloadedStrings
+  - RecordWildCards
+  - ScopedTypeVariables
+  - StandaloneDeriving
+  - TemplateHaskell
+  - TupleSections
+  - TypeApplications
+  - TypeFamilies
+  - ViewPatterns
```
Comment (DanielRrr) [inline .stylish-haskell.yaml:61]: I believe one may remain all those comment above

Comment (pgujjula) [reply .stylish-haskell.yaml:61]: Sorry, I don't understand. What do you mean by "remain all those comment"?

Comment (DanielRrr) [reply .stylish-haskell.yaml:61]: Sorry, I meant "delete", not "remain". I'm half-asleep now.

Comment (pgujjula) [reply .stylish-haskell.yaml:61]: Fixed.

**Pull request merged**